.github/workflows/ci.yml

```diff
@@ -62,48 +70,20 @@ jobs:
       DEBUG: ipfsd-ctl:* # enables all debug output from javascript 'debug' lib used by js-ipfsd-ctl
 
     runs-on: ${{ matrix.platform.host }}
+    name: ${{ matrix.platform.name }}
     steps:
     - name: Checkout sources
       uses: actions/checkout@v2
 
-    - name: Cache cargo folder
-      uses: actions/cache@v2
-      with:
-        path: |
-          ~/.cargo/registry
-          ~/.cargo/git
-        key: ${{ matrix.platform.target }}-dot-cargo-parts-${{ hashFiles('Cargo.lock') }}
-
-    - name: Get npm cache directory
-      id: npm-cache
-      if: matrix.platform.cross == false
-      run: |
-        echo "::set-output name=dir::$(npm config get cache)"
-
-    - name: Cache npm (non-cross targets)
-      uses: actions/cache@v2
-      if: matrix.platform.cross == false
-      with:
-        path: ${{ steps.npm-cache.outputs.dir }}
-        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json', '**/setup.sh', '**/*.patch') }}
-        restore-keys: |
-          ${{ runner.os }}-node-
-
-    - name: Install dependencies ubuntu
+    - name: Install dependencies (linux)
       if: matrix.platform.host == 'ubuntu-latest'
       run: sudo apt-get install llvm-dev libssl-dev pkg-config
 
-    - name: Install dependencies macos
+    - name: Install dependencies (macos)
       if: matrix.platform.host == 'macos-latest'
       run: brew install llvm openssl
 
-    - name: Download go-ipfs on Linux
-      if: matrix.platform.host == 'ubuntu-latest'
-      run: |
-        curl -L https://github.com/ipfs/go-ipfs/releases/download/v0.7.0/go-ipfs_v0.7.0_linux-amd64.tar.gz --output go_ipfs.tar.gz
-        tar -xf go_ipfs.tar.gz
-
-    - name: Install dependencies windows (openssl)
+    - name: Install dependencies (windows)
```

Comment:
There's still the nagging about "setupOnly" which ... I don't know why it would interfere with caching. https://github.com/rs-ipfs/rust-ipfs/pull/471/checks?check_run_id=3339830233#step:33:3

Reply:
Ok, this is luckily nicely documented, I just had missed it: https://github.com/marketplace/actions/run-vcpkg#setup-vcpkg-only-and-use-your-own-scripts

Reply:
Just enabled this option, will test if it caches on my fork

Reply:
This still requires the openssl installing step.